04-reporting.Rmd

```diff
@@ -510,17 +509,176 @@ kableExtra: https://cran.r-project.org/web/packages/kableExtra/index.html
+- `full_result`: combines this information.
+- `table`: arranges model terms, estimates and test statistics in a `data.frame`
+
+List elements `$estimate`, `$statistic`, and `$full_result` are intended to be
+included directly into the body of your manuscript by using inline code chunks.
+For instance, you could first save the result from `apa_print()` in an output object
+
+```{r}
+out <- apa_print(
+  t.test(yield ~ N, data = npk)
+)
+```
+
+and then include a single list element by writing:
+
+```{r inline-results, eval = FALSE, highlight = FALSE}
+Plant yield indeed differed between groups, `r out$full_result`.
+```
+
+The above line is rendered as follows:
+
+> Plant yield indeed differed between groups, `r out$full_result`.
+
+
+The last element `table` is intended to be used to create results tables that may be included in the manuscript via `apa_table()`
+It will be most useful for ANOVA or linear-model results, as you will see in the next subsections.
+
+### Reporting models and tests in a table
+
+For statistical models with multiple terms (for instance, multiple regression or multi-way ANOVA) it is sometimes convenient
+to report the statistical tests of all model terms in a table.
+For this purpose, `apa_print()` returns the list element `$table` that may be included using function `apa_table()` or any other table
+(for instance, `kable()` from the **knitr** package).
+Consider the following regression model
+
+```{r}
+lm_out <- lm(formula = Post_QoL ~ Base_QoL + BDI, data = cosmetic_surgery)
+summary(lm_out)
+```
+
+To include the model table into your manuscript, first apply `apa_print()` to the model object,
+and then use `apa_table()` on the list element `$table`.
+
+```{r}
+apa_lm <- apa_print(lm_out)
+apa_table(
+  apa_lm$table
+  , caption = "A full regression table."
+)
+```
+
+
+
+
+### Fully supported analysis methods
+
+There is a plethora of possible output objects from statistical analyses in R,
+and it is of course impossible to support all of them.
+However, over the course of time, we were able to create a corpus of methods that will likely
+cover many experimental psychologists' toolboxes.
+
+- All NHST that are delivered with the base R `stats` package, which includes:
+    - $t$ for means, $\chi^2$, and nonparametric equivalents `t.test()`
+    - tests for correlation coefficients `cor.test()`
+    - (generalized) linear models
+    - ...
+    - except for pairwise comparisons (these should be done with the package **emmeans**)
+- Analysis of Variance ANOVA and ANCOVA (from packages **stats**, **car**, and **afex**)
+- Multivariate Analysis of Variance MANOVA and MANCOVA (from `stats`)
+- Hierachical (Generalized) Linear Models (from **lme4**, **lmerTest**, **nlme**, and **afex**)
+- Pairwise comparisons and planned contrasts from **emmeans**, **lsmeans**, and **glht**
+- Bayesian *t* tests and ANOVAs from **BayesFactor**
```

Comment:
Could you note here that this implementation is "experimental"?

Reply:
done